### PR TITLE
feat(docs): move readme plugins to wiki

### PR DESCRIPTION
**Pull request by theol0403**
Moves readme to wiki, to allow people to contribute with tips, configurations, and to reduce bloat in the readme.

### CODE REVIEW

**Comment by theol0403** [comment]
I've been considering keeping "Getting Started" in the readme - what do people think?

**Comment by justinmk** [comment]
The readme  should be made less verbose and only mention useful information. Part of that can be achieved by removing some of the hacks and questionable features from vscode-neovim.

But , 👎  for removing all of this content from the readme. Wiki is less discoverable and less controllable. Readme is a good place to bring the most important info front-and-center. Also I think it drives the vscode marketplace page?

**Comment by theol0403** [comment]
> The readme should be made less verbose and only mention useful information. Part of that can be achieved by removing some of the hacks and questionable features from vscode-neovim.
> 
> But , -1 for removing all of this content from the readme. Wiki is less discoverable and less controllable. Readme is a good place to bring the most important info front-and-center. Also I think it drives the vscode marketplace page?

Fair enough, I agree with these points. I do think the readme should be condensed, but I don't think that information should disappear for interested users. I have condensed some things in this PR.

I think the latest commit brings a decent balance - all important information is in the readme, and details/tips are in the wiki.

Reasoning for putting in wiki:
- plugins: people can share tips on how to make popular plugins work best with vscode-neovim. This should not really be part of the readme.
- bindings: these are large tables that should be on a separate page. Unfortunately they are useful because vscode works differently for files/windows/tabs, so this plugin rebinds almost all ex-commands and behaviour varies.
- tips: I can move these to the readme if you think I should, but this can contain configuration tips, advanced documentation, and troubleshooting

Let me know if tips should be moved to the readme.

Either way, I think this is an improvement, because 1. the community can be more involved with sharing tips (making a PR adds friction), 2. the important information in the readme is more visible.

**Comment by theol0403** [comment]
Alright, I've moved it back, but hopefully I managed to organize the readme a bit better and condense it a bit.

File: README.md [hunk 1]
```diff
@@ -237,9 +235,8 @@ write `:w` or `:noh`). <kbd>Up</kbd>/<kbd>Down</kbd> selects the option and <kbd
 
 Multiple cursors work in:
 
-1. Insert mode
-2. Visual line mode
-3. Visual block mode
+1. Insert mode 2 isual line mode
```
Comment (justinmk) [inline README.md:238]: typos

File: README.md [hunk 2]
```diff
@@ -253,7 +250,7 @@ To spawn multiple cursors from visual line/block modes type <kbd>ma</kbd>/<kbd>m
 
 See gif in action:
 
-![multicursors](/images/multicursor.gif)
+![multicursors](https://github.com/vscode-neovim/vscode-neovim/blob/master/images/multicursor.gif)
```
Comment (justinmk) [inline README.md:253]: the relative form is useful for local markdown renderers

File: README.md [hunk 3]
```diff
@@ -326,23 +308,81 @@ Open definition aside (default binding):
...
+These are the default commands and bindings available for file/scroll/window/tab management.
 
--   See [vscode-scrolling.vim](/vim/vscode-scrolling.vim) for scrolling commands reference
--   See [vscode-file-commands.vim](/vim/vscode-file-commands.vim) for file commands reference
--   See [vscode-tab-commands.vim](/vim/vscode-tab-commands.vim) for tab commands reference
--   See [vscode-window-commands.vim](/vim/vscode-window-commands.vim) for window commands reference
+-   See [vscode-scrolling.vim](https://github.com/vscode-neovim/vscode-neovim/blob/master/vim/vscode-scrolling.vim) for
+    scrolling commands reference
+-   See
+    [vscode-file-commands.vim](https://github.com/vscode-neovim/vscode-neovim/blob/master/vim/vscode-file-commands.vim)
```
Comment (justinmk) [inline README.md:326]: is there some autoformatter doing this?

Comment (theol0403) [reply README.md:326]: Yeah, I had some issues with the autoformatter messing up with things, thanks for catching

**Review by justinmk — APPROVED** [review]

**Pull request merged**